src/patternfly/components/Nav/nav.scss

```diff
@@ -183,7 +183,8 @@
   --pf-c-nav__toggle--FontSize: var(--pf-global--icon--FontSize--md);
 
   // Toggle icon
-  --pf-c-nav__toggle-icon--Transition: var(--pf-global--TransitionDuration);
+  --pf-c-nav__toggle--icon--Transition: var(--pf-global--TransitionDuration);
```

Comment:
since we don't have a wrapper that is pf-c-nav__toggle-icon anymore should this just be `pf-c-nav__toggle` ?